test/Microsoft.AspNetCore.SignalR.Redis.Tests/RedisServerFixture.cs

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.SignalR.Tests.Common;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Testing;
+
+namespace Microsoft.AspNetCore.SignalR.Redis.Tests
+{
+    public class RedisServerFixture<TStartup> : IDisposable
+        where TStartup : class
+    {
+        public ServerFixture<TStartup> FirstServer { get; private set; }
+        public ServerFixture<TStartup> SecondServer { get; private set; }
+
+        private readonly ILogger _logger;
+        private readonly ILoggerFactory _loggerFactory;
+        private readonly IDisposable _logToken;
+
+        public RedisServerFixture()
+        {
+            if (Docker.Default == null)
```

Comment:
Add some comment that it means that Docker is not installed and the tests will be skipped.